### PR TITLE
ASA and SeaBASS news items

### PR DESCRIPTION
**Pull request by aditya-uw**
@leewujung I added 3 news items for ASA school, ASA, and SeaBASS. 

Let me know if you have any suggestions, I'll also look at the preview and let you know what I think again actually

### CODE REVIEW

**Comment by aditya-uw** [comment]
@leewujung okay I think it looks good from my end, any suggestions?

File: content/newslist.dat [hunk 1]
```diff
@@ -1,8 +1,13 @@
+**[06/2024]** Wu-Jung lectured and Aditya attended [SeaBASS in New Hampshire](https://eos.unh.edu/center-acoustics-research-education/education/bioacoustic-summer-school-seabass) and met some inspiring lecturers and students!
```
Comment (leewujung) [inline content/newslist.dat:1]:
```suggestion
**[06/2024]** Aditya attended the [BioAcoustic Summer School (SeaBASS) in University New Hampshire](https://eos.unh.edu/center-acoustics-research-education/education/bioacoustic-summer-school-seabass) and met some inspiring lecturers and students! Wu-Jung also gave a lecture on Fundamentals of Ocean Acoustics!

```

File: content/newslist.dat [hunk 2]
```diff
@@ -1,8 +1,13 @@
+**[06/2024]** Wu-Jung lectured and Aditya attended [SeaBASS in New Hampshire](https://eos.unh.edu/center-acoustics-research-education/education/bioacoustic-summer-school-seabass) and met some inspiring lecturers and students!
+
+**[05/2024]** Wu-Jung and Aditya presented two talks in the [Ottawa ASA Meeting](https://acousticalsociety.org/ottawa/) on our HakeML and UBNA passive acoustic monitoring projects.
+
+**[05/2024]** Wu-Jung provided a lecture at [ASA School in Ottawa](https://acousticalsociety.org/asa-school-2024/) as an instructor and met some wonderful students!
```
Comment (leewujung) [inline content/newslist.dat:5]:
```suggestion
**[05/2024]** Wu-Jung gave a lecture on active acoustic ocean sensing at [ASA School in Ottawa](https://acousticalsociety.org/asa-school-2024/) as an instructor and met some wonderful students!

```

**Review by leewujung — APPROVED** [review]
Thanks @aditya-uw! Minor tweaks and I think this is good to merge!

**Pull request merged**